src/Slider/Slider.jsx

```diff
@@ -648,6 +698,9 @@ const Slider = class Slider extends React.Component {
       ...filteredTrayProps
     } = trayProps;
 
+    const infiniteBefore = infinite ? React.cloneElement(children[children.length - 1], { key: "infiniteBefore" }) : null;
```

Comment:
This is placing the last element at the beginning before the first element. A couple of questions:
- How does this impact if people are starting the carousel at the beginning? Would it now start at index 0 which is the cloned element?
- This is only cloning the last element or first element, not "next" element. So if someone goes back, and then back again, it doesn't load the _next_ slide?
- Does _not_ changing the totalSlides property to match the totalSlides available, does this break anything else in the component?